src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(../src/fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+}
+
+input{
```

Comment:
It is not recommended to apply styles directly to the HTML tags. It's better to use classes for styling.